init_venv.sh

```diff
@@ -89,26 +82,37 @@ else
 fi
 
 CONSTRAINTS_FILE=$(tempfile)
+export PIP_CONSTRAINT=${CONSTRAINTS_FILE}
```

Comment:
For my understanding, where is this variable used?

Reply:
Didn't find documentation but it works as global constraints for pip: https://stackoverflow.com/a/60639466